packages/perseus/src/widgets/dropdown/dropdown.tsx

```diff
@@ -74,13 +75,27 @@ class Dropdown extends React.Component<Props> implements Widget {
                 key="placeholder"
                 value="0"
                 disabled
-                label={this.props.placeholder || " "}
+                label={
+                    <Renderer
+                        content={this.props.placeholder}
+                        strings={this.context.strings}
+                        inline
+                    />
```

Comment:
Using `<Renderer>` feels heavy for rendering TeX. Perhaps using the `<TeX>` component would be a lighter-weight implementation? [Access to this component is via `getDependencies()`](https://github.com/Khan/perseus/blob/341d316aa8727ebb9e7fde28fc4e2d8779aa3e82/packages/perseus/src/components/sortable.tsx#L777) instead of direct import. Can we give that a try?

Reply:
Looks like that component always renders everything as TeX.

<img width="468" alt="Screenshot 2024-12-17 at 2 34 05 PM" src="https://github.com/user-attachments/assets/a43b3d4a-d459-401a-975c-b44dc2a41301" />

Reply:
Bummer. OK. Let's stick with the `<Renderer>` widget for now. Thanks for checking into it.

Reply:
Mark and I chatted in DMs, but I'll circle back here.  I suggested the `TeX` component, but didn't realize there might be a mixture of standard content and TeX markup. In that case, `Renderer` is the correct tool. It handles parsing content and rendering TeX as needed. This is pretty common to use the `Renderer` in widgets to render content (even if widgets aren't supported).